### PR TITLE
Add org.kde.kbreakout

### PR DESCRIPTION
**Pull request by cho2**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of [permissions](https://invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.kbreakout.json#L10). 
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
  - I am a member of [KDE Team on Flathub](https://github.com/orgs/flathub/teams/kde/members).
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - KDE project owns the domain
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 
  - https://invent.kde.org/games/kbreakout/-/merge_requests/2

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by cho2** [comment]
bot, build org.kde.kbreakout

File: org.kde.kbreakout.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kbreakout",
+    "rename-icon": "kbreakout",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host"
```
Comment (hfiguiere) [inline org.kde.kbreakout.json:12]: for a game?

also don't you need dri?

File: org.kde.kbreakout.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "id": "org.kde.kbreakout",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kbreakout",
+    "rename-icon": "kbreakout",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.kbreakout.json:10]:
```suggestion
        "--socket=fallback-x11",
```

if it supports wayland

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by cho2** [comment]
bot, build org.kde.kbreakout

File: org.kde.kbreakout.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "id": "org.kde.kbreakout",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kbreakout",
+    "rename-icon": "kbreakout",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=-fallbackx11",
```
Comment (hfiguiere) [inline org.kde.kbreakout.json:10]:
```suggestion
        "--socket=-fallback-x11",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.kbreakout.json (hunk 3 above)
Comment (cho2) [reply org.kde.kbreakout.json:10]: oops, typo

**Comment by cho2** [comment]
```
flatpak run org.kde.kbreakout
Qt: Session management error: Could not open network socket
libGL error: MESA-LOADER: failed to retrieve device information
QOpenGLFramebufferObject: Framebuffer incomplete attachment.
QOpenGLFramebufferObject: Framebuffer incomplete, missing draw buffer.
QOpenGLFramebufferObject: Framebuffer incomplete, missing draw buffer.
QOpenGLFramebufferObject: Framebuffer incomplete, missing draw buffer.
QOpenGLFramebufferObject: Framebuffer incomplete, missing attachment.
QOpenGLShader: could not create shader
QOpenGLShader: could not create shader
QOpenGLShader: could not create shader
intel_do_flush_locked failed: error Input/Output
```

**Comment by cho2** [comment]
bot, build org.kde.kbreakout

**Comment by cho2** [comment]
cc @flathub/kde

**Comment by albert-astals-cid-kdab** [comment]
Looks reasonable

**Comment by tsdgeos** [comment]
Looks reasonable

**Comment by travier** [comment]
Minor comment. LGTM

**Comment by cho2** [comment]
bot, build org.kde.kbreakout

**Comment by cho2** [comment]
cc @barthalion @nedrichards

**Pull request closed without merging**